scripts/src/main/resources/scripts/command/lazydocker

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env bash
+
+# autocompletion list
+if [ "${1}" = "shortlist" ]
+then
+  if [ -z "${2}" ]
+  then
+    echo "setup help"
+  fi
+  exit
+fi
+
+if [ -n "${DEVON_IDE_TRACE}" ]; then set -vx; fi
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
+
+# Call lazydocker with specified arguments.
+function doRun() {
+  doSetup silent
+  cd "${LAZYDOCKER_HOME}" || exit 1
+  if doIsWindows
+  then
+    ldInPathRes=$(echo "$PATH" | tr ":" "\n" | grep lazydocker)
+    if [ -z "${ldInPathRes}" ]
+    then
+      export PATH=${PATH}:"${LAZYDOCKER_HOME}"
+    fi
+    powershell.exe -command 'start powershell {lazydocker}'
+  else
+    ./lazydocker
```

Comment:
so what I actually wanted to say is the you do not need to do `cd ${LAZYDOCKER_HOME}` and can just call `lazydocker` without the `./` what also makes the assumption that it has to be directly in `software/lazydocker` and we get that content from an external source - maybe in future releases they decide to move the binary to a `bin` subfolder.